tests/FunctionAwaitAllTest.php

```diff
@@ -34,8 +34,10 @@ public function testAwaitAllRejected()
         Block\awaitAll($all, $this->loop);
     }
 
-    public function testAwaitAllRejectedWithFalseWillWrapInUnexpectedValueException()
+    public function testAwaitAllRejectedWithExceptionWillWrapInUnexpectedValueException()
     {
+        $this->skipForPromise3();
```

Comment:
I see why you've decided to add a helper method for this, but I wonder if it could be more descriptive if we added a reason each time? What do you think? See also all following occurrences.

Reply:
Added a `reason` parameter for the function to pass through when skipping the test